.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  push:
+    paths:
+      - '.github/workflows/test.yml'
+      - 'src/*'
+      - 'profiling/*'
+      - 'benchmarks/*'
+
+jobs:
+  test_cpython:
+    name: CPython ${{ matrix.py }} for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+        py: [3.5, 3.6, 3.7]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install Rust
```

Comment:
I'm curious if there is a "rustup" action already. If not it would be a nice addition to the Github marketplace.

Reply:
not that I could see, but I suspect it can't be more of a hassle than the setup-python action. I'll check it out (in Vegas though, don't expect it soon)